docs/pages/docs/apis/fields.mdx

```diff
@@ -137,11 +137,15 @@ A `checkbox` field represents a boolean (`true`/`false`) value.
 
 Options:
 
-- `defaultValue` (default: `undefined`): Can be either a boolean value or an async function which takes an argument `({ context, originalInput })` and returns a boolean value.
-  This value will be used for the field when creating items if no explicit value is set.
-  `context` is a [`KeystoneContext`](./context) object.
-  `originalInput` is an object containing the data passed in to the `create` mutation.
-- `isRequired` (default: `false`): If `true` then this field can never be set to `null`.
+- `defaultValue` (default: `false`): This value will be used for the field when creating items if no explicit value is set.
+- `graphql.read.isNonNull` (default: `false`): If you have no read access control and you don't intend to add any in the future,
+  you can set this to true and the output field will be non-nullable. This is only allowed when you have no read access control becaue otherwise,
```

Comment:
```suggestion
  you can set this to true and the output field will be non-nullable. This is only allowed when you have no read access control because otherwise,
```